src/lib/fetcher.ts

```diff
@@ -4,8 +4,9 @@ import { RequestOptions } from './requestOptions';
 import * as querystring from 'querystring';
 import events = require('events');
 import urlJoin = require('url-join');
-
-import fetch = require('isomorphic-fetch');
+if (navigator.product !== 'ReactNative') {
```

Comment:
Hm, odd, navigator must only be available in certain contexts, despite [this](http://stackoverflow.com/a/9590297). FWIW, my PR works in RN.